backend/domain/creation.go

```diff
@@ -4,3 +4,16 @@ type TextReq struct {
 	Text   string `json:"text" validate:"required"`
 	Action string `json:"action"` // action: improve, summary, extend, shorten, etc.
 }
+
+// FIM (Fill in Middle) tokens
+const (
```

Comment:
FIM token常量的定义是合理的，但建议在注释中说明这些token的具体用途和来源，以便其他开发者理解其作用。

```suggestion
// FIM (Fill in Middle) tokens - 用于指示模型在文本中间进行填充
const (
    FIMPrefix = "<fim_prefix>"  // 前缀标记
    FIMSuffix = "<fim_suffix>"  // 后缀标记
    FIMMiddle = "<fim_middle>"  // 中间填充标记
)